R120 notes: chromeos_startup.sh does not exist in R120, this patch is invalid

--- a/init/chromeos_startup.sh
+++ b/init/chromeos_startup.sh
@@ -776,6 +776,8 @@
   fi
 fi
 
+. /usr/share/cros/fydeos_factory_install.sh
+
 bootstat post-startup
 
 # Always return success to avoid killing init
